complex/ft_config.c

```diff
@@ -669,7 +669,7 @@ void fts_cur_info(struct ft_series *series, struct ft_info *info)
 	info->class_function = set->class_function[series->cur_func];
 	info->op = set->op[series->cur_op];
 	info->test_flags = set->test_flags;
-	info->caps = set->caps[series->cur_caps];
+	info->caps = set->caps[series->cur_caps] | FT_CAP_MSG;
```

Comment:
I'm trying to find the easiest way of dealing with this.  That seems to be to push it to the config files, which isn't friendly.  For now, I wonder if we can tack on MSG when reading the config file, since the test implementations require them.